5.0/en/0x51-V51-OAuth2.md

```diff
@@ -82,7 +84,7 @@ Restricting token privileges ensures a Client is granted the proper access to a
 
 ### Resource Owner Password Credentials Grant
 
-Aside from this grant type can leak credentials in more places than just the Authorization Server, adapting the Resource Owner password credentials grant to two-factor authentication, authentication with cryptographic credentials (e.g. WebCrypto, WebAuthn) and authentication processes that require multiple steps can be hard or impossible.
+Aside from this grant type can leak credentials in more places than just the Authorization Server, adapting the Resource Owner password credentials grant to two-factor authentication, authentication with cryptographic credentials (e.g. WebCrypto, WebAuthn) and authentication processes that require multiple steps can be hard or impossible. This grant type is not recommended in general due to security concerns. Instead, use the authorization code grant with PKCE. This grant type is omitted from OAuth 2.1 specification. 
```

Comment:
> Instead, use the authorization code grant with PKCE. This grant type is omitted from OAuth 2.1 specification.

Is is me or might this be interpreted as "authorization code grand with PKCE is omitted from OAuth 2.1"?

Reply:
Maybe swap the two sentences?

Reply:
Hi @randomstuff, great catch! Now that I read it a few times, it makes sense to swap the positioning of sentences. I'll update this PR as well for this.